### DISCUSSION TITLE
Why Nepal is not listed in 2FA?

### DISCUSSION THREAD

**Discussion by mrjudiyt** [community/community#108924] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I'm from Nepal and recently a 2FA notification popped up on my GitHub profile and I tried to do so. Then when I tried to create 2FA through SMS my country was not mentioned. Please consider adding my country code.
My country is `Nepal(+977)
<img width="928" alt="263617003-7dad75e9-3c47-41db-b62b-96912dd0070e" src="https://github.com/community/community/assets/91169292/80b74001-d844-4493-a472-466ab1ae451c">

**Comment by queenofcorgis** [comment]
If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).
